src/components/table/table.scss

```diff
@@ -97,6 +97,15 @@ $tabulator-arrow-color-active: rgb($tabulator-arrow-color-active-raw-value);
     .tabulator-cell {
         border-right: transparent;
         padding: functions.pxToRem(8);
+
+        display: inline-flex;
```

Comment:
I am not sure whether we should mark this as a breaking change or not. I know this will affect for instance one of our own components as described here: https://github.com/Lundalogik/crm-feature/issues/2447

Reply:
Hmm… yeah… that's a good question… 🤔

It's better to err on the side of caution, so I'd say yes, mark it as breaking, and describe the change and potential effect in the breaking change description 👍

Reply:
Thanks, I just don't know the rituals of introducing breaking changes? where do I write that this is a breaking change? How to mark it?

Reply:
https://github.com/Lundalogik/lime-elements/blob/next/CONTRIBUTING.md#footer 🙂